donut/modules/editor/static/script.js

```diff
@@ -26,7 +26,7 @@ function preview() {
       converter = new showdown.Converter({strikethrough: true, tables: true}),
       html = converter.makeHtml(text);
   target.html(html);
-  target_title.text(title);
+  target_title.text(title.replace("_", " "));
```

Comment:
JS replace only replaces the first occurrence by default; should use `/_/g` to replace all